crates/bevy_winit/src/system.rs

```diff
@@ -0,0 +1,245 @@
+use bevy_ecs::{
+    entity::Entity,
+    event::EventReader,
+    prelude::{Added, Changed, With},
+    system::{Commands, NonSendMut, Query, Res},
+};
+use bevy_utils::tracing::{error, info};
+use bevy_window::{
+    Cursor, CursorPosition, PrimaryWindow, Window, WindowClosed, WindowComponents, WindowHandle,
+    WindowMode, WindowPosition, WindowResizeConstraints, WindowResolution, WindowState,
+    WindowTitle,
+};
+use raw_window_handle::HasRawWindowHandle;
+use winit::{
+    dpi::{LogicalSize, PhysicalPosition},
+    event_loop::EventLoopWindowTarget,
+};
+
+use crate::{converters, get_best_videomode, get_fitting_videomode, WinitWindows};
+
+/// System responsible for creating new windows whenever a `Window` component is added
+/// to an entity.
+///
+/// This will default any necessary components if they are not already added.
+pub fn create_window_system(
```

Comment:
```suggestion
pub fn create_window(
```
probably don't need the "system"